docs/policy/dynamic-pools.md

```diff
@@ -0,0 +1,122 @@
+# Dynamic-Pools Policy
+
+## Overview
+
+The dynamic-pools policy puts the workload in the dynamic-pool that are disjoint CPU pools. The dynamic-pools can be resized, that is CPUs added and removed, which is based on the requests of the containers and the CPU utilization of the workload in the dynamic-pools.
+
```

Comment:
The dynamic-pools policy can put the workload into different dynamic-pools. Each dynamic-pool contains several CPUs and can be resized dynamically in terms of the specific algorithms.

Reply:
Thanks for your suggestions!
I have updated the dynamic-pools policy document, please see the latest PR.